test/auth.test.js

```diff
@@ -0,0 +1,96 @@
+var auth = require('http-auth');
```

Comment:
Not sure. I didn't look into it because I assumed the tests were node-only, and http-auth is only used for the tests.